kura/test/org.eclipse.kura.nm.test/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
What is the purpose of this file?

Reply:
This is needed to mock static classes, as presented [here](https://www.baeldung.com/mockito-mock-static-methods).